packages/demo-webpack/README.md

```diff
@@ -8,4 +8,4 @@ import { createWidget } from "@typeform/embed";
 createWidget("<form id>", { container: document.querySelector("#container") });
 ```
 
-Use `yarn start` to run the demo in browser: http://localhost:8080
+Run `yarn build` in [embed package](../embed). Then run `yarn start` to run the demo in browser: http://localhost:8080
```

Comment:
Add section title `Example`
And maybe instructions to clone this specific part of the repo, to just have this example?

Anything about common issues with SSR and webpack?
Should we add a section about how to handle this?

Reply:
SSR will "just work", no workaround necessary.